tests/test_sampling.py

```diff
@@ -110,21 +110,21 @@ def test_linear_regression(self, case, is_mass_matrix_diagonal):
 normal_test_cases = [
     {
         "algorithm": blackjax.hmc,
-        "initial_position": jnp.array(100.0),
+        "initial_position": jnp.array(3.0),
         "parameters": {
-            "step_size": 0.1,
-            "inverse_mass_matrix": jnp.array([0.1]),
-            "num_integration_steps": 100,
+            "step_size": 3.9,
+            "inverse_mass_matrix": jnp.array([1.0]),
+            "num_integration_steps": 30,
         },
         "num_sampling_steps": 6000,
-        "burnin": 5_000,
+        "burnin": 1_000,
     },
     {
         "algorithm": blackjax.nuts,
-        "initial_position": jnp.array(100.0),
-        "parameters": {"step_size": 0.1, "inverse_mass_matrix": jnp.array([0.1])},
+        "initial_position": jnp.array(3.0),
+        "parameters": {"step_size": 4.0, "inverse_mass_matrix": jnp.array([1.0])},
         "num_sampling_steps": 6000,
-        "burnin": 5_000,
+        "burnin": 1_000,
```

Comment:
Could you put these changes to a separate PR?

Reply:
I feel like a separate commit is enough.